### PR TITLE
removed the ride complete button

### PR DESCRIPTION
**Pull request by jwpeterkim**
## Description

What does the PR do?
Removes the unnecessary complete button in the RideView page.

## Background Information
Complete ride button is redundant since dropping off the rider automatically completes the ride and we don't want to say the ride is completed unless the rider is dropped off.

## Screenshots
Screen Shot 2021-02-11 at 23 43 56

## Checklist:
- [v] Screenshots attached (if dealing with UI)
- [ ] Unit Tests (if applicable)

## Testing Steps for reviewer:
- Step 1 
Go to RideView.js and see the lines 269-289 and 554-560 being commented out. 
- Step 2
Check if the modal doesn't show the complete ride button on your simulator. (it should now show only the ride cancel button.)

### CODE REVIEW

File: app/views/RideView/RideView.js [hunk 1]
```diff
@@ -266,27 +266,27 @@ export default class RideView extends Component {
...
+  //       console.log('dropping off res', result);
+  //       API.completeRide(rideId, token).then(result => {
+  //         console.log('complete ride request', result);
+  //         this.setState({ visible: false }, () => {
+  //           navigation.navigate('MainView', { completed: true });
+  //         });
+  //       });
+  //     });
+  //   });
+  // };
```
Comment (kidatsy) [inline app/views/RideView/RideView.js:289]: Just 🪓 this all!

File: app/views/RideView/RideView.js [hunk 2]
```diff
@@ -551,13 +551,13 @@ export default class RideView extends Component {
...
                     />
-                    <Button
+                    {/* <Button
                       title="Complete Ride"
                       containerStyle={styles.startRideContainer}
                       titleStyle={styles.modalTitle}
                       buttonStyle={styles.modalButton}
                       onPress={() => this.completeRideAlert()}
-                    />
+                    /> */}
```
Comment (kidatsy) [inline app/views/RideView/RideView.js:560]: ✂️

**Review by kidatsy — CHANGES_REQUESTED** [review]
Just cut the code out, instead of commenting!

**Comment by jwpeterkim** [comment]
> Just cut the code out, instead of commenting!

Me and Elsa just weren't sure if we'd be needing or referencing some of "the complete ride button" codes in the future. Especially since it looked like there were lots of work put into it. But if you can confirm we won't need them, I can definitely just remove them and resubmit it! Let me know what you think!

**Comment by jwpeterkim** [comment]
Hi TC, I removed unnecessary lines as we discussed. It should be good to go!
Let me know if there are any other issues with this PR. Thanks!

**Review by kidatsy — APPROVED** [review]
LGTM!

**Pull request merged**